src/Ddeboer/Imap/Message/Part.php

```diff
@@ -154,7 +154,7 @@ public function getDecodedContent()
                     break;
 
                 case self::ENCODING_QUOTED_PRINTABLE:
-                    $this->decodedContent =  \utf8_encode(\quoted_printable_decode($this->getContent()));
+                    $this->decodedContent =  \iconv($this->getCharset(), 'UTF-8', \quoted_printable_decode($this->getContent()));
```

Comment:
Let’s use `mb_convert_encoding()` instead, as we’re already using that elsewhere in this library.